apps/photos/src/types/file/index.ts

```diff
@@ -5,21 +5,17 @@ import {
 } from 'types/magicMetadata';
 import { Metadata } from 'types/upload';
 
-interface FileAttributesBase {
-    decryptionHeader: string;
-}
-
-interface MetadataFileAttributes extends FileAttributesBase {
+export interface MetadataFileAttributes {
     encryptedData: string;
+    decryptionHeader: string;
     objectKey?: string;
 }
-interface S3FileAttributes extends FileAttributesBase {
+export interface S3FileAttributes {
     objectKey: string;
     encryptedData?: string;
```

Comment:
I think it's safe to remove `encryptedData` from `S3FileAttributes` and `objectKey` from `MetadataFileAttributes`. But keeping it to be on safer side for now.

Reply:
Okay, just saw @abhinavkgrd's comment. Will go ahead and remove